monitoring/mongodb/alerts.test.yaml

```
@@ -13,78 +13,84 @@ tests:
   - name: MongoDbDown
     interval: 30s
     input_series:
-      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-mongos", pod="data-db-mongodb-sharded-mongos-0"}
+      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-shard0-data", pod="data-db-mongodb-sharded-shard0-data-0"}
         values: 1 1 0 0 0 0 1 1 1
-      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-mongos", pod="data-db-mongodb-sharded-mongos-1"}
+      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-shard0-data", pod="data-db-mongodb-sharded-shard0-data-1"}
         values: 1 1 1 1 0 0 0 1 1
-      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-mongos", pod="data-db-mongodb-sharded-mongos-2"}
+      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-shard0-data", pod="data-db-mongodb-sharded-shard0-data-2"}
         values: 1 1 1 1 1 0 0 0 1
+      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-shard0-data", pod="data-db-mongodb-sharded-shard1-data-0"}
+        values: 1x9
+      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-shard0-data", pod="data-db-mongodb-sharded-shard1-data-1"}
+        values: 1x9
+      - series: up{namespace="zenko", job="zenko/data-db-mongodb-sharded-shard0-data", pod="data-db-mongodb-sharded-shard1-data-2"}
+        values: 1x9
+      - series: kube_statefulset_replicas{namespace="zenko", statefulset="data-db-mongodb-sharded-shard0-data", job="kube-state-metrics"}
+        values: 3x9
+      - series: kube_statefulset_replicas{namespace="zenko", statefulset="data-db-mongodb-sharded-shard1-data", job="kube-state-metrics"}
+        values: 3x9
```

Is this list missing a third one for shard2 ?
```suggestion
        - series: kube_statefulset_replicas{namespace="zenko", statefulset="data-db-mongodb-sharded-shard2-data", job="kube-state-metrics"}
        values: 3x9
```

---

no nvm